### ISSUE TITLE
[audiobookbay] Status and Updates

### ISSUE THREAD

**Issue by garfield69** [Jackett/Jackett#15444]
Id: audiobookbay
Name: AudioBook Bay
Description: AudioBook Bay (ABB) is a Public Torrent Tracker for AUDIOBOOKS
SiteLink: https://audiobookbay.lu/
AlternativeSiteLinks:
	https://audiobookbay.lu/
	http://audiobookbay.is/
	http://audiobookbay.se/
	http://audiobookbay.fi/
	http://audiobookbay.ws/
	http://theaudiobookbay.se/

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
https://audiobookbay.is/ ERR_CONNECTION_TIMED_OUT

**Comment by hv33y** [comment]
It's back again.

**Comment by garfield69** [comment]
Hmm. for selected regions only. I'm still getting the timeout.

**Comment by hv33y** [comment]
Yes, you're right - [uptime](https://uptime.com/freetools/global-uptime-test/1111053/audiobookbay.is)

**Comment by hv33y** [comment]
Moreover, the official website is [audiobookbay.lu](https://audiobookbay.lu/). mentioned at [r/AudioBookBay](https://www.reddit.com/r/AudioBookBay/).

**Comment by garfield69** [comment]
I've added the alternate domain to the indexer, but uptime shows its DNS coverage is the same as the other domain.

**Comment by hv33y** [comment]
> I've added the alternate domain to the indexer, but uptime shows its DNS coverage is the same as the other domain.

yea ikr period

**Comment by garfield69** [comment]
v0.22.563

**Comment by mosfet** [comment]
A bit of info, I can search audiobook bay but only if the request begins with a lower case letter. I suspected this when, on the audiobookbay website, it lowercases your search with javascript before it submits.

Tested this in the manual search, and realized the same:

![image](https://github.com/user-attachments/assets/e1304c28-295e-489c-980e-5c72c16f2ed8)


vs

![image](https://github.com/user-attachments/assets/405566fe-dd93-4c8c-927c-7e52af8c7d2f)


In more testing, this turns out to be only the first character being lowercase:

![image](https://github.com/user-attachments/assets/2dc7c99c-ee02-4aa1-9343-e6cfa7cf3681)

**Comment by garfield69** [comment]
v0.22.664

**Comment by garfield69** [comment]
both domains are now available to all uptime checkpoints apart from newdelhi

**Comment by garfield69** [comment]
https://audiobookbay.lu/ > timeout
uptrends has 41 checkpoints offline with TCP Connection Failed (timeout)

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://audiobookbay.lu/ > TCP Connection Failed (Times out)
http://audiobookbay.is/ > TCP Connection Failed (Times out)
http://audiobookbay.se/ > TCP Connection Failed (Times out)
http://audiobookbay.fi/ > TCP Connection Failed (Times out)
http://audiobookbay.ws/ > TCP Connection Failed (Times out)
http://theaudiobookbay.se/ > TCP Connection Failed (Times out)

**Comment by garfield69** [comment]
only http://audiobookbay.is/  remains offline, the rest are now online again.
